pkg/restore/log_client.go

```diff
@@ -343,7 +344,7 @@ func (l *LogClient) writeToTiKV(ctx context.Context, kvs kv.Pairs, region *Regio
 	lastKey := codec.EncodeBytes([]byte{}, kvs[len(kvs)-1].Key)
 
 	meta := &sst.SSTMeta{
-		Uuid:        uuid.NewV4().Bytes(),
+		Uuid:        []byte(uuid.New().String()),
```

Comment:
```suggestion
		Uuid:        uuid[:]
```

The field should contain 16 bytes, not the xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx string.